system-addon/content-src/components/Sections/_Sections.scss

```diff
@@ -39,14 +39,14 @@
         background-size: 50px 50px;
         -moz-context-properties: fill;
         display: block;
-        fill: $fill-secondary;
+        fill: var(--newtab-icon-secondary-color);
         height: 50px;
         margin: 0 auto;
         width: 50px;
       }
 
       .empty-state-message {
-        color: $text-secondary;
+        color: var(--newtab-text-primary-color);
```

Comment:
For default theme, the text is no longer faded, but I think we don't want it too light anyway for contrast…

Reply:
so, leave it or?